src/deck/guides/konnect.md

```diff
@@ -0,0 +1,227 @@
+2. decK configuration file (default: `~/.deck.yaml`)
+3. File passed with the `--konnect-password-file` flag
+
+For example, if you have both a decK config file and a {{site.konnect_short_name}} password file, decK uses the password in the config file.
+
+### Authenticate with CLI flags
+
+There are two ways to authenticate through CLI flags: by passing the password in `--konnect-password` directly, or by saving it to a file and passing the filename to decK with `--konnect-password-file`.
+
+With `konnect-password`:
+
+```sh
+deck ping \
+  --konnect-email YOUR_EMAIL \
+  --konnect-password YOUR_PASSWORD
+```
+
+Or, save your {{site.konnect_short_name}} password to a file, then pass the file using the `--konnect-password-file` flag:
+
+```sh
+deck ping \
+  --konnect-email YOUR_EMAIL \
+  --konnect-password-file /PATH/TO/FILE
+```
+
+### Authenticate with a decK config file
+
+The default decK configuration file is `~/.deck.yaml`.
+You can store {{site.konnect_short_name}} credentials in this file to pass them more securely.
+For example:
+
+```
+konnect-email: example@email.com
+konnect-password: YOUR_PASSWORD
+```
+
+Alternatively, you can save your password in a separate file, then specify the password file instead of a password:
+
+```
+konnect-email: example@example.com
+konnect-password-file: PATH/TO/FILENAME
+```
+
+decK automatically uses the credentials in `~/.deck.yaml` in any subsequent calls:
+
+```
+deck ping
+
+Successfully Konnected as MyName (Konnect Org)!
+```
+
+## Target a {{site.konnect_short_name}} API
+
+Use `--konnect-addr` to select the API to connect to.
+
+The default API decK uses is `https://us.api.konghq.com`, which targets the `cloud.konghq.com` environment.
+If your account is in this environment, you don't need to change anything.
+
+If your account is in the `konnect.konghq.com` environment, use this flag to target the relevant API:
+
+```sh
+deck ping --konnect-addr https://konnect.konghq.com
+```
+
+## Runtime groups
+
+Target runtime groups in your state file with the `konnect_runtime_group` parameter:
+
+```yaml
+_format_version: "1.1"
+_konnect:
+  runtime_group_name: staging
+```
+
+You can also set this parameter using the `--konnect-runtime-group-name` flag:
+
+```sh
+deck sync --konnect-runtime-group-name default
+```
+
+A state file can only target one runtime group.
+
+If you leave this empty, don't provide a flag, or don't include a `_konnect` section at all, decK targets the `default` runtime group.
+You can see this by pushing a config file to {{site.konnect_short_name}} with `deck sync`.
+
+1. Create a basic config file:
+
+    ```yaml
+    _format_version: "1.1"
+    services:
+    - name: example_service
+      host: mockbin.org
+    ```
+
+2. Sync the file to {{site.konnect_short_name}}:
+
+    ```sh
+    deck sync
+    ```
+
+3. Pull down the configuration with `deck dump`:
+
+    ```sh
+    deck dump --konnect-runtime-group-name default
+    ```
+
+    See the newly added `_konnect` section:
```

Comment:
```suggestion
    See the newly added `_konnect` section. For example:
```
One question I had is why they need to see the new section. Do they need to use that info in another spot?

Reply:
In hindsight, this entire section is irrelevant. It's literally just here to illustrate what happens if you don't provide a group, but there's no reason to document that with steps.